core/extension/index.js

```diff
@@ -5,167 +5,236 @@
 
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis
 const globalSymbol = Symbol.for("itwin.core.frontend.globals");
-const ext = globalThis[globalSymbol].getExtensionApi(import.meta.url);
+const ext = globalThis[globalSymbol].getExtensionApi("import.meta.url");
```

Comment:
@johnnyd710 what issues did you run into with using `import.meta.url`? Is the plan to skip it for this PR and get it working later?

Reply:
The plan is to tackle it in another PR. Using `import.meta.url` gives error `Module parse failed: Unexpected token` when trying to add build extensions to the iTwin Viewer. Didn't want that to get in the way of this PR

Reply:
Okay, sounds good.